### ISSUE TITLE
Generate hash version based on HEAD pointer

### ISSUE THREAD

**Issue by ciotlosm** [danielwelch/hassio-zigbee2mqtt#29]
See https://github.com/Koenkk/zigbee2mqtt/pull/125. When this is merged, we should use a similar pattern as https://github.com/ciotlosm/zigbee2mqtt/blob/b6aabcca6443d66c2b3de7cc646aab51e3331779/tag.sh to generate a hash.

**Comment by danielwelch** [comment]
Sorry for my inactivity recently! I’m in the process of moving across about half the United States. Could you summarize what the plan is for the versioning story in zigbee2mqtt? Looks like there will be a versioned release, and this issue specifically is for displaying a version to the user on startup, in the event that they’re not using a stable release.

**Comment by ciotlosm** [comment]
No worries. Always fun to break routine 

Versioning for zigbee2mqtt is getting closer to 0.1 (actually 0.2 because currently it's version 0.1 in package.json). However the setup I pointed out is very useful in a few cases:
- When people do a rollback from HASSIO addon by using the SHA feature
- When people will be running "cutting edge" master version as we discussed

The feature should be very easy to implement and lightweight, so I see quite a bit of value in adding the implementation.
